### ISSUE TITLE
Incorrect network usage (specifically Device rx)

### ISSUE THREAD

**Issue by sergi0g** [corecoding/Vitals#414]
### Has this issue been covered in the Wiki?

- [X] I have read the Wiki and my issue is not covered.

### Is there an existing issue reported already?

- [X] I have searched the existing issues, it does not exist.

### What is the unexpected behavior?

`Device rx` is constantly showing `3.7 MB/s` (it augments when using the network of course), but I am not using the network. I have checked with System Monitor and `lsof -i` and also terminated processes to debug. My network usage is just a few KB/s.

It seems that the number is stuck, because running another `gnome-shell` session (`dbus-run-session -- gnome-shell --nested --wayland`) shows the proper network usage.

### Steps to reproduce the unexpected behavior.

You probably can't reproduce this issue.

### What distribution and version of Linux are you using?

Ubuntu 23.10

### What version of Gnome are you using?

45.2

**Comment by corecoding** [comment]
The Device rx/tx is the sum of all of your network devices. Can you provide a screenshot of your entire network menu? I would assume there is something adding up to create that 3.7MB/s. It is possible there is an interface that Vitals should ignore, but we'll have to identify what that is in order to submit a fix.

**Comment by hex-developer** [comment]
Thank you for responding! I do not have a screenshot of the menu, but I did check the rest of the network menu myself too. I saw the unusual network usage (my network speed caps at 3 MB/s, so I couldn't believe it when I saw 6) after downloading something from within a VM, so it's possible Vitals may have been using a virtual interface. However, after destroying the VM (and the interfaces it used), Vitals would still report 3.7 MB/s. If I encounter the problem again I will notify you. Thanks for your awesome extension!
